src/Record.ts

```diff
@@ -225,7 +236,16 @@ export function map<A, B>(fa: Record<string, A>, f: (a: A) => B): Record<string,
 }
 
 /**
+ * Reduce object by iterating over it's values.
+ *
  * @since 1.10.0
+ *
+ * Warning: If your operation isn't commutative and the record's keys don't have consistent order you may get inconsistent results
```

Comment:
> the record's keys don't have consistent order

What do you mean?

Reply:
see the example

Reply:
See also
https://github.com/ramda/ramda/issues/2046

Reply:
> `reduce({a: 1, b: 2}, f) != reduce({b: 2, a: 1}, f)`

@jethrolarson Actually they are equal since keys are sorted https://github.com/gcanti/fp-ts/blob/7c090c5d0ee7d1257f56035b87351626e19bfa73/src/Record.ts#L276

Reply:
I could swear I looked for that. Okay. I will revise

Reply:
Updated